### PR TITLE
Death of the Mining Scanner

### PR DESCRIPTION
**Pull request by Ergovisavi**
- Sets mining turfs back to fully visible ore at all times
- Removes scanning functionality from Mining Scanner. It's just for Gibtonite and Mining Bots again.
- Removes the Mech Mining Scanner / Adv scanner from the code.

Once upon a time, Mining was changed to be atmospheric, dark, and spooky. Mesons had their full-lit tiles removed, and the tunnels were dark and filled with monstrosities. The mining scanner was necessary to find ore in the darkness. With mesons reverted, the mining scanner completely lost its purpose, but nobody removed it, they just kept "improving" it by making it automatic for some reason. Without the darkness, the mining scanner was a completely pointless feature, other than making the asteroid tolerable to look at.

This is the future you chose. One of an asteroid that looks like a rainbow, but you chose it.

RIP

### CODE REVIEW

**Comment by xxalpha** [comment]
:+1:

**Comment by GunHog** [comment]
:+1: Happiness!

File: code/modules/mining/equipment_locker.dm [hunk 1]
```diff
@@ -778,65 +775,6 @@
 	slot_flags = SLOT_BELT
 	var/cooldown = 0
```
Comment (Fox-McCloud) [inline code/modules/mining/equipment_locker.dm]: doesn't need a CD anymore, I don't believe.

**Comment by Ergovisavi** [comment]
This is being closed for investigation into other... things.

**Pull request closed without merging**